go.sum

```diff
@@ -78,5 +78,3 @@ maunium.net/go/mauflag v1.0.0 h1:YiaRc0tEI3toYtJMRIfjP+jklH45uDHtT80nUamyD4M=
 maunium.net/go/mauflag v1.0.0/go.mod h1:nLivPOpTpHnpzEh8jEdSL9UqO9+/KBJFmNRlwKfkPeA=
 maunium.net/go/maulogger/v2 v2.3.1 h1:fwBYJne0pHvJrrIPHK+TAPfyxxbBEz46oVGez2x0ODE=
 maunium.net/go/maulogger/v2 v2.3.1/go.mod h1:TYWy7wKwz/tIXTpsx8G3mZseIRiC5DoMxSZazOHy68A=
-maunium.net/go/mautrix v0.10.6 h1:hQAoo9mKODQvIGiYZIZfW14GFo0fwEOiMIKrZ72OvGE=
-maunium.net/go/mautrix v0.10.6/go.mod h1:k4Ng5oci83MEbqPL6KOjPdbU7f8v01KlMjR/zTQ+7mA=
```

Comment:
These should've been replaced with the new version checksums, `replace`s need to be commented out while running `go mod tidy` (this might get better with go module workspaces in 1.18)

Reply:
Oh, that's annoying. I was wondering why it got removed.